src/Microsoft.DotNet.Darc/Darc/Operations/GetAssetOperation.cs

```diff
@@ -46,11 +46,14 @@ public override async Task<int> ExecuteAsync()
 
             // Starting with the remote, get information on the asset name + version
             List<Asset> matchingAssets =
-                (await remote.GetAssetsAsync(name: _options.Name, version: _options.Version)).ToList();
+                (await remote.GetAssetsAsync(name: _options.Name, version: _options.Version, buildId: _options.Build)).ToList();
 
             string queryDescriptionString =
-                $"name '{_options.Name}'{(!string.IsNullOrEmpty(_options.Version) ? $" and version '{_options.Version}'" : "")}" +
-                $"{(targetChannel != null ? $" on channel '{targetChannel.Name}'" : "")} in the last {_options.MaxAgeInDays} days";
+                $"name '{_options.Name}'" +
```

Comment:
Style preference - This mix of interpolated strings, concatenation and inclusion of string.Empty looks like we are expressing the whole logic in string. Suggest to use string builder and add the filters (on channel) etc. only if the condition is met. It will express the logic more naturally than this.

Reply:
Sure, copilot made the change